supportedBackends/google/pipelines/common/src/test/scala/cromwell/backend/google/pipelines/common/PipelinesApiRuntimeAttributesSpec.scala

```diff
@@ -279,6 +279,18 @@ class PipelinesApiRuntimeAttributesSpec extends WordSpecLike with Matchers with
     }
   }
 
+  "should parse cpuPlatform correctly" in {
+    val runtimeAttributes = Map("docker" -> WomString("ubuntu:latest"))
+    val workflowOptionsJson =
+      """{
+        |  "default_runtime_attributes": { "cpuPlatform": "the platform" }
+        |}
+      """.stripMargin.parseJson.asInstanceOf[JsObject]
+    val workflowOptions = WorkflowOptions.fromJsonObject(workflowOptionsJson).get
+    val expectedRuntimeAttributes = expectedDefaults.copy(cpuPlatform = Some("the platform"))
```

Comment:
We generally `Option` rather than `Some` to construct option instances and handle nulls in the expected way (generating `None`s rather than `Some(null)`s. Obviously for a literal that's not a concern but probably a good habit anyway. 🙂